@here/harp-theme-tools/src/cli-build-theme.ts

```diff
@@ -54,11 +55,50 @@ inputFiles.forEach(file => {
         resolveIncludeUris: true
     }).then(theme => {
         theme.url = undefined;
-        const filename = `${cliOptions.out}/${path.basename(file)}`;
-        console.log(`Writing ${filename}`);
         const json = cliOptions.minify
             ? JSON.stringify(theme)
             : JSON.stringify(theme, undefined, 4);
-        fs.writeFileSync(filename, json);
+
+        const unifiedDefinitionsFile =
```

Comment:
"unifiedDefinitionsFile" should be "unifiedDefinitionsFileName"

Reply:
fixed